BLSQ/iaso
# WC2-124 dependencies update

Update critical dependencies:

![Screenshot 2022-12-15 at 14 49 23](https://user-images.githubusercontent.com/12494624/207875690-088f6d04-8518-46bc-9b07-38ca51127760.png)
to

![Screenshot 2022-12-15 at 16 01 26](https://user-images.githubusercontent.com/12494624/207894518-09643683-e610-4268-9632-94449f0c56a9.png)


Also import of video.js css was missing and video was looking like this:

![Screenshot 2022-12-15 at 10 58 03](https://user-images.githubusercontent.com/12494624/207876301-bfc29d04-058a-4827-8831-5a4b2f4e9691.png)


## Self proof reading checklist

## Changes

- remove `react-leaflet-control` by removing use of Control in `MarkerMapComponent`
- update: @formatjs/cli, babel-loader, css-loader, style-loader, jsdom, node-sass to non breaking changes versions
- import back video.js css to make it look better

## How to test

Remove node_modules and run` npm ci` or delete docker webpack container and run a new build.

Launch the frontend and check dashboard is loading.
Go to an instance detail with a location and check the layer selection on the map is present and working.
You can add a mp4 file to an existant instance, then go to the list of submissions filtered on the form from the instance were you added a video, go to files, video, and check the render of the video

## Print screen / video

<img width="990" alt="Screenshot 2022-12-15 at 12 29 07" src="https://user-images.githubusercontent.com/12494624/207877332-38e6e427-0d79-4be4-83db-b5f8e7269083.png">
<img width="547" alt="Screenshot 2022-12-15 at 14 57 12" src="https://user-images.githubusercontent.com/12494624/207877541-4b0c8a58-4ea9-48fb-8f57-847aad6bac7e.png">
